src/pages/HomePage.jsx

```diff
@@ -1,45 +1,40 @@
-import clsx from 'clsx'
 import { useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { Link } from 'react-router-dom'
-import { useMedia } from 'react-use'
+import Button from '../components/Button/Button'
 
 const HomePage = () => {
   const [query, setQuery] = useState('')
   const { t } = useTranslation()
-  const isMobile = useMedia('(max-width: 599px)')
 
   const onChange = event => {
     setQuery(event.target.value)
   }
 
   return (
-    <div className="homepage">
-      <div className="homepage__wrapper">
-        <h1 className="homepage__heading">{t('FIND_YOUR_NEW_FLAT')}</h1>
+    <div className="homepageBackground">
+      <div className="homepage">
```

Comment:
Kebab-case only. To keep the BEM structure:
```suggestion
    <div className="homepage">
      <div className="homepage__container">
```